src/app/layout/account-switcher.component.html

```diff
@@ -72,17 +71,17 @@
           <span class="server" aria-hidden="true" *ngIf="a.value.serverUrl != 'bitwarden.com'">{{
             a.value.serverUrl
           }}</span>
-          <span class="status" aria-hidden="true">{{ a.value.profile.authenticationStatus }}</span>
+          <span class="status" aria-hidden="true">{{
+            (a.value.profile.authenticationStatus === authStatus.Unlocked ? "unlocked" : "locked")
+              | i18n
+          }}</span>
         </div>
         <i
-          class="bwi bwi-unlock bwi-2x text-muted"
-          aria-hidden="true"
-          *ngIf="a.value.profile.authenticationStatus == 'unlocked'"
-        ></i>
-        <i
-          class="bwi bwi-lock bwi-2x text-muted"
+          class="bwi bwi-2x text-muted"
+          [ngClass]="
+            a.value.profile.authenticationStatus == authStatus.Unlocked ? 'bwi-unlock' : 'bwi-lock'
```

Comment:
No real change here, I just find `ngClass` to be more succinct when swapping between icons.